README.md

```diff
@@ -65,12 +65,22 @@ Need help? This is a developer's portal for CoBlocks and should not be used for
 
 ## Contributions
 
-Please read the [guidelines for contributing](https://github.com/godaddy/coblocks/blob/master/.github/CONTRIBUTING.md) to CoBlocks. Anyone is welcome to contribute. If you find a 🐞 or an issue, [create an issue](https://github.com/godaddy/coblocks/issues/new). Here are ways to contribute:
+Thank you for thinking about contributing to CoBlocks plugin! If you're unsure of anything, know that you're 💯 welcome to submit an issue or pull request on any topic. The worst that can happen is that you'll be politely directed to the best location to ask your question or to change something in your pull request. We appreciate any sort of contribution and don't want a wall of rules to get in the way of that.
+
+Before contributing, please read the [guidelines for contributing](https://github.com/godaddy/coblocks/blob/master/.github/CONTRIBUTING.md) to CoBlocks. Anyone is welcome to contribute. If you find a 🐞 or an issue, [create an issue](https://github.com/godaddy/coblocks/issues/new). Here are ways to contribute:
 
 1. Raise an [issue](https://github.com/godaddy/coblocks/issues/new)
 2. Send a pull request with your bug fixes and/or new features
 3. Provide feedback and suggestions
 
+## First Time Contributor?
+
+New to open source/free software? Here is a selection of issues we've made **especially for first-timers**.
+[Good first issues](https://github.com/godaddy/coblocks/labels/good%20first%20issue)
+
+We're here to help, so just raise  an [issue](https://github.com/godaddy/coblocks/issues/new) and ask if one looks interesting or if you have questions.
```

Comment:
Looks like 2 spaces between `raise` and `an`

Reply:
Good catch!

Reply:
fixed.